src/Administration/Resources/app/administration/src/core/consent/consent.store.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * @private
+ * @sw-package framework:fundamentals
+ */
+export type ConsentDTO = {
+    readonly name: string,
+    readonly identifier: string,
+    status: 'accepted' | 'revoked' | 'requested',
+}
+
+type ConsentStoreState = {
+    consents: Record<string, ConsentDTO>;
+}
+
+/**
+ * @private
+ */
+export default Shopware.Store.register('consent',{
+    state: (): ConsentStoreState => ({
+        consents: {},
```

Comment:
Should that be an array?

Reply:
The last I've seen from your api is that you return an associated array from the `list` method. This would transform to an object in JS